spec/interactor/initializer_spec.rb

```diff
@@ -106,4 +106,52 @@ def run
       end
     end
   end
+
+  context 'with bare inclusion' do
+    let(:interactor) do
+      Class.new do
+        include Interactor::Initializer
+
+        def run
+          :result
+        end
+      end
+    end
+
+    describe '.run' do
+      subject { interactor.run }
+
+      it { is_expected.to eq(:result) }
+    end
+
+    describe '.for' do
+      subject { interactor.for }
+
+      it { is_expected.to eq(:result) }
+    end
+
+    describe '.with' do
+      subject { interactor.with }
+
+      it { is_expected.to eq(:result) }
+    end
+
+    context 'when class is not including interactor initializer' do
+      let(:interactor) do
+        Class.new do
+          def run
+            :result
+          end
+        end
+      end
+
+      describe '.run' do
+        subject { interactor.run }
+
+        it 'cannot call interactor' do
+          expect { subject }.to raise_error(/undefined method/)
+        end
+      end
+    end
```

Comment:
To me this spec seems to be testing Ruby itself, is it useful to have it here? If I understand correctly this would pass without any changes to the `Interactor`, right?

Reply:
Yes, the omitted part of `include interactor...` makes it fail. I added it later because I thought it would be more clear of how it is expected that it works and that it's not a coincidence.